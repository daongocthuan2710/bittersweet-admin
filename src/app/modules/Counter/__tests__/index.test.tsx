import { render } from '@testing-library/react';
import Test from '../components/Test';

describe('Basic DatePicker Component', () => {
  it('should render the Basic Test Component', () => {
    const { container } = render(<Test />);
    expect(container).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
