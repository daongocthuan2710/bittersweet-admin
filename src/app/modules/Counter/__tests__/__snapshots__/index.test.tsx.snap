// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Basic DatePicker Component should render the Basic Test Component 1`] = `
<div>
  <div>
    Test something
  </div>
</div>
`;
